problems/objects.js

```diff
@@ -141,7 +175,15 @@ const addsCountry = () => {};
  * @returns {Object} {"Mark Twain": 8.9, "Nathaniel Hawthorne": 5.1}
  */
 
-const authorScores = () => {};
+const authorScores = (authors) => {
+  let obj = {}
+  for(let authorsArr of authors){
+    const name = authorsArr[0]
```

Comment:
try not to make const variables inside of a loop, because you'll be re-assigning the value so technically it is changing